source/opt/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ add_library(SPIRV-Tools-opt
   types.h
   unify_const_pass.h
   value_number_table.h
+        workaround1209.h
```

Comment:
formatting looks funny

Reply:
Fixed.  The formater in the IDE messes up the cmake files every time.  I think it is because it adds a tab.